### ISSUE TITLE
research github releases

### ISSUE THREAD

**Issue by boundedinfinity** [boundedinfinity/userdotd#6]
I'd like to figure out a way to use the something like [travisci](https://travis-ci.org) to to create [github release](https://help.github.com/articles/creating-releases).

Possibly building on the work from the [userdotd: Up CI](https://github.com/boundedinfinity/userdotd/issues/1) issue.

**Comment by berryhill** [comment]
[possible solution](https://github.com/dominique-mueller/automatic-release)

**Comment by boundedinfinity** [comment]
Yes, looks like it has potential, I'd like to investigate to see if there is something already built into github itself to do this, but if not, then we can use what ever.

**Comment by boundedinfinity** [comment]
You can see that the official github page has a [Automatically creating releases](https://help.github.com/articles/creating-releases/#automatically-creating-releases) section.  Just haven't had the time to dig into it.

**Comment by berryhill** [comment]
@boundedinfinity ok, will take a look, down with whatever works at ease.

**Comment by berryhill** [comment]
@boundedinfinity ok, will take a look, down with whatever works at ease.
